ledger-service/http-json/src/main/scala/com/digitalasset/http/json/JsonProtocol.scala

```diff
@@ -385,12 +385,15 @@ object JsonProtocol extends JsonProtocolLow {
         val reference: JsObject =
           ContractLocatorFormat.write(obj.reference).asJsObject("reference must be an object")
 
-        val fields: Vector[(String, JsValue)] =
-          reference.fields.toVector ++
-            Vector("choice" -> obj.choice.toJson, "argument" -> obj.argument.toJson) ++
-            obj.meta.cata(x => Vector("meta" -> x.toJson), Vector.empty)
-
-        JsObject(fields: _*)
+        val fields =
+          reference.fields ++
+            Iterable("choice" -> obj.choice.toJson, "argument" -> obj.argument.toJson) ++
```

Comment:
Why are we using `Iterable` here and `Seq` in `ledger-service/http-json/src/test/scala/com/digitalasset/http/json/JsonProtocolTest.scala` a bit below ?

Reply:
Good point, the other one should be `Iterable` too.

Reply:
f3781ede45d